rfd/0058-package-distribution.md

```diff
@@ -0,0 +1,66 @@
+---
+authors: Fred Heinecke (fred.heinecke@goteleport.com)
+state: draft
+---
+
+# RFD 58 - Package Distribution
+
+## What
+
+Hosting and configuration of APT and YUM repositories.
+
+## Why
+
+Currently we are building .deb and .rpm file in Teleport's Drone pipeline via reprepro and createrepo, then publishing to AWS S3. Between the tools used and the pipeline's current configuration there are two main issues: a lack of channel support for major versions, and an inability to host multiple minor versions for each major version. By fixing these issues we can allow customers to upgrade the teleport package to the latest minor version release for their major version, as well as allow them to roll back to a previous minor version if need be.
+
+## Details
+
+Several potential solutions were investigated and their features compared as shown below:
```

Comment:
I'd put the matrix of alternatives considered below the recommended plan.  Why? In the long run, the recommendation and technical details are what most folks will refer to this document for -- the research will fade in importance over time.

Reply:
Makes sense. RFD updated.